javascript-loops-practice/get-indexes.js

```diff
@@ -0,0 +1,10 @@
+/* exported getIndexes */
+function getIndexes(array) {
+  const newArr = [];
+  for (const ind in array) {
+    newArr.push(Number(ind));
+  }
```

Comment:
`for..in` loops are for objects. You don't want to use `for..in` loops with arrays. They work, because arrays are technically objects, but you should be using a `for` loop instead